### DISCUSSION TITLE
Get a list of projects of which a specified user or group is a member

### DISCUSSION THREAD

**Discussion by thw0rted** [goharbor/harbor#19417] [Q&A]
I'm a server admin, and would like to use the API to get a list of non-public projects that a particular user or `usergroup` (OIDC) would see when they log in.  I believe I can look up membership each individual project (`/projects/1/members`, `/projects/2/members` etc) but of course this isn't going to scale well on an instance with many projects.  Is there a faster way?  (If not, would this be an appropriate feature request?)

**Comment by stonezdj** [comment]
No such API so far.
